.github/workflows/integration_tests.yaml

```diff
@@ -59,6 +59,10 @@ jobs:
         run: sudo apt remove moby-runc
       - name: Install rename
         run: sudo apt install rename
+      - name: Install and start journald
+        run: |
+          sudo apt-get update && sudo apt-get install -y systemd
```

Comment:
Let's use only one tool to manage packages - **apt**

Reply:
okay